.github/workflows/release.yml

```diff
@@ -0,0 +1,89 @@
+# This action handles the automatic publishing of new releases
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-package:
+    name: Publish package
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - run: npm ci
+      - run: |
+          echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > .npmrc
+          cat .npmrc
+        # We do not create a commit with the update package version
+        # Therefore the package.*.json in the repository will always have out-of-sync versions
```

Comment:
Not blocking for this PR, but we need to use the new standardised way of referencing the token in npmrc

Reply:
oh sure, I was not aware a new standardised way. Of course it'll be better

Reply:
We decided it together 🤦

Reply:
oh THAT new standardised way xD